api/src/test/java/org/openmrs/api/AdministrationServiceTest.java

```diff
@@ -443,23 +439,21 @@ public void saveGlobalProperty_shouldFailIfGivenAllowedLocaleListDoesNotContainD
 		assertThat("localeList contains default locale but should not for this test case", localeList,
 			not(containsString(LocaleUtility.getDefaultLocale().toString())));
 
-		expectedException.expect(APIException.class);
-		expectedException.expectMessage("can not be removed from allowed locales list because it is the default locale");
 		
-		adminService.saveGlobalProperty(
-			new GlobalProperty(OpenmrsConstants.GLOBAL_PROPERTY_LOCALE_ALLOWED_LIST, localeList));
+		APIException exception = assertThrows(APIException.class , () -> adminService.saveGlobalProperty(
+			new GlobalProperty(OpenmrsConstants.GLOBAL_PROPERTY_LOCALE_ALLOWED_LIST, localeList)));
+		assertThat(exception.getMessage(), containsString(" can not be removed from allowed locales list because it is the default locale"));
```

Comment:
```suggestion
		assertThat(exception.getMessage(), containsString("can not be removed from allowed locales list because it is the default locale"));
```